CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - A changelog file.
 
+### Changed
+
+- Require swap-cli user to specify the connection details of the peer they wish to swap with.
```

Comment:
```suggestion
- Require the `swap` CLI user to specify the connection details of the peer they wish to swap with.
```

Reply:
what about buyer instead of swap-cli user?